hack/capi-sync-provider.sh

```
@@ -0,0 +1,102 @@
+PROVIDER="${1:?usage: capi-sync-provider.sh <provider>}"
+ROOT="$(cd "$(dirname "$0")/.." && pwd)"
+MOD_DIR="$ROOT/hack/capi-vendor/$PROVIDER"
+DST_DIR="$ROOT/pkg/capi/$PROVIDER"
+COPY_TOOL="$ROOT/hack/copy-capi-types/main.go"
+BOILERPLATE="$ROOT/hack/boilerplate.go.txt"
+CONTROLLER_GEN="${CONTROLLER_GEN:-controller-gen}"
+
+GO="${GO:-GO111MODULE=on GOWORK=off GOFLAGS=-mod=vendor go}"
+
+run_go() {
+    eval "$GO" "$@"
```

Shellcheck (0.11.0)

[warning] eval negates the benefit of arrays. Drop eval to preserve whitespace/symbols (or eval as string).

(SC2294)